.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.{kt,kts}]
+disabled_rules=import-ordering
```

Comment:
I disabled checking import ordering because it has always been kind of annoying especially that in Android Studio we don't have any easy way to reorder imports so that ktlint is satisfied. And now with new updates they changed something and I was asked to change the order of menu imports despite the fact everything was fine before.

Reply:
Is there no way to fix using "Import layout" in Android Studio > Preferences > Editor > Code Style > Kotlin? It would be nice to keep ordering enforced, so we don't end up causing accidental changes in files using "optimize imports".

Reply:
Ok I downgraded ktlint because they might have something broken regarding the ordering of imports in the newest version if what we have on master is no longer accepted. I even tried different settings like:
```
ij_kotlin_imports_layout=* # alphabetical with capital letters before lower case letters (e.g. Z before a), no blank lines
ij_kotlin_imports_layout=*,java.**,javax.**,kotlin.**,^ # default IntelliJ IDEA style, same as alphabetical, but with "java", "javax", "kotlin" and alias imports in the end of the imports list
```
to no avail.